### DISCUSSION TITLE
Toast notifications on Mobile

### DISCUSSION THREAD

**Discussion by elibroftw** [unoplatform/uno#12145] [Q&A]
How to show toast notifications for Mobile?

**Comment by jeromelaban** [comment]
Can you specify which platform you need toast notifications for?

**Reply by elibroftw** [reply]
Mobile

**Reply by pkar70** [reply]
Android? iOS? Symbian? :)

**Reply by jeromelaban** [reply]
Thanks, but which ones specifically? At this time, we do not have support for toast on any mobile targets (Aside from https://github.com/unoplatform/uno/pull/2585). If you want to send toast notifications, you can use the native platforms APIs instead.

**Reply by elibroftw** [reply]
Android for now and then later iOS so both of them. The last time I did mobile development was in 2020 with Flutter, so I don't exactly know how to use native platform APIs. Would it be C# still or Java/Kotlin?

**Reply by wartron** [reply]
Im too looking for a little toast bubble or something for when an action happens, main use case is for a button that copies a string to the clipboard, and saying "copied to clipboard", targeting mobile and win10

**Reply by jeromelaban** [reply]
@wartron for this particular scenario, you can use the `InfoBar` control.
